### PR TITLE
Update razor.md

### PR DESCRIPTION
**Pull request by Rick-Anderson**
[Review URL](https://review.docs.microsoft.com/en-us/aspnet/core/mvc/views/razor?branch=pr-en-us-3839#razor-reserved-keywords)

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@ryandle  quick review please.

**Comment by Rick-Anderson** [comment]
@ryandle quick review please.

**Comment by ryandle** [comment]
hey @Rick-Anderson I think you mentioned the wrong person, did you want @rynowak ?

**Comment by Rick-Anderson** [comment]
@ryandle  sorry about that - no wonder @rynowak  isn't responding. I'll try not to do that again. 
@rynowak  please review.

File: aspnetcore/mvc/views/razor.md [hunk 1]
```diff
@@ -584,4 +588,5 @@ Set a break point on the `Compile` method of `CustomCompilationService` and view
 ![Text Visualizer view of compilationContent](razor/_static/tvr.png)
 
 > [!NOTE]
-> View lookups are case sensitive. If your controller routing seeks a view named `Index` (*Index.cshtml*) but you've named your view file `index` (*index.cshtml*), you'll receive an exception: `InvalidOperationException: The view 'Index' was not found.`
+> View lookups are case sensitive. If your controller routing seeks a view named `Index` (*Index.cshtml*) but you've named your view file `index` (*index.cshtml*), you'll receive an exception: `InvalidOperationException: The view 'Index' was not found.` 
+[Areas](xref:mvc/controllers/areas) are case sensitive. `[Area("admin")]` will not work if your folder is named *Areas/Admin*.
```
Comment (rynowak) [inline aspnetcore/mvc/views/razor.md]: I'm almost certain that this is not the case. @pranavkm - can you come up with something more accurate that describes everything users need to know about casing + views?

**Comment by Rick-Anderson** [comment]
@pranavkm Awesome info. Minor edit of your work. Ok to merge now?

**Pull request merged**